qiskit/schemas/examples/qobj_openpulse_example.json

```
@@ -33,7 +33,8 @@
             {"name": "acquire", "t0": 10, "duration": 90, "qubits": [0,1,2], "register_slot": [0,1,2], "memory_slot": [0,1,2]},
             {"name": "copy", "register_orig": 1, "register_copy": [3,4]},
             {"name": "bfunc", "mask": "0xE", "relation": "==", "val": "0xA", "register": 3, "memory": 3},
-            {"name": "pulse1", "ch": "d0", "t0": 120, "conditional": 3}
+            {"name": "pulse1", "ch": "d0", "t0": 120, "conditional": 3},
+            {"name": "parametric_pulse", "t0": 0, "params": {"amp": [0.0, 0.5], "sigma": 4, "duration": 25}, "ch": "d0", "pulse_shape": "gaussian"},
```

```suggestion
            {"name": "parametric_pulse", "t0": 0, "params": {"amp": [0.0, 0.5], "sigma": 4, "duration": 25}, "ch": "d0", "pulse_shape": "gaussian"}
```

Small suggestion for the `json.decoder.JSONDecodeError: Expecting value: line 38 column 13 (char 1594)` in current travis run - json is a bit picky when it comes to commas after the last element of an array.